manifests/b/BinaryFortress/CloudShow/6.5.0.0/BinaryFortress.CloudShow.installer.yaml

```diff
@@ -1,18 +1,12 @@
-# Created using wingetcreate 1.9.3.0
+# Created with YamlCreate.ps1 v2.4.4 $debug=QUSU.CRLF.7-5-0.Win32NT
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
 
 PackageIdentifier: BinaryFortress.CloudShow
 PackageVersion: 6.5.0.0
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
 InstallerType: inno
 Scope: machine
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
 UpgradeBehavior: install
+ProductCode: 7fb59370-6c93-47cd-91b7-c3eb75e76cd7_is1
 Installers:
 - Architecture: x86
   InstallerUrl: https://binaryfortressdownloads.com/Download/BFSFiles/128/CloudShowLauncherSetup-6.5.exe

```

Comment:
```suggestion
```
The following line, with `InstallerSha256`, needs to be removed also. But the review tool won't let me suggest that line also.